packages/components/src/navigation/utils.js

```diff
@@ -0,0 +1,11 @@
+/**
+ * External dependencies
+ */
+import { deburr } from 'lodash';
+
+// @see packages/block-editor/src/components/inserter/search-items.js
```

Comment:
Would it worth opening an issue to extract this as a util function? So we can reuse it in both places.

Reply:
Yes definitely!
My intention was to open a follow up right after merging this, and this is a nice reminder, since at this point I'm sure I'd have forgot about it. 😄